flight/targets/naze32/board-info/board-info.mk

```diff
@@ -0,0 +1,25 @@
+BOARD_TYPE          := 0xA0
```

Comment:
This needs to added to https://github.com/TauLabs/TauLabs/wiki/Development-Hardware-IDs